src/manifest.c

```diff
@@ -1075,34 +1014,8 @@ struct list *consolidate_files(struct list *files)
 	 * "list" and "next" point to the first and second in a series of perhaps
 	 * many objects referring to the same filename.  As we determine which file out
 	 * of multiples to keep in our consolidated, deduplicated, filename sorted list
-	 * there are Manifest invariants to maintain.  The following table shows the
-	 * associated decision matrix.  Note that "file" may be a file, directory or
-	 * symlink.
-	 *
-	 *         | File 2:
-	 *         |  A'    B'    C'    D'
-	 * File 1: |------------------------
-	 *    A    |  -  |  2  |  2  |  2  |
-	 *    B    |  1  |  -  |  2  |  2  |
-	 *    C    |  1  |  1  |  -  |  X  |
-	 *    D    |  1  |  1  |  X  |  X  |
-	 *
-	 *   State for file1 {A,B,C,D}
-	 *         for file2 {A',B',C',D'}
-	 *       A:  is_deleted && !is_rename
-	 *       B:  is_deleted &&  is_rename
-	 *       C: !is_deleted && (file1->hash == file2->hash)
-	 *       D: !is_deleted && (file1->hash != file2->hash)
-	 *
-	 *   Action
-	 *       -: Don't Care   - choose/remove either file
-	 *       X: Error State  - remove both files, LOG error
-	 *       1: choose file1 - remove file2
-	 *       2: choose file2 - remove file1
-	 *
-	 * NOTE: the code below could be rewritten to be more "efficient", but clarity
-	 *       and concreteness here are of utmost importance if we are to correctly
-	 *       maintain the installed system's state in the filesystem across updates
+	 * there are Manifest invariants to maintain.
+         * Note that "file" may be a file, directory or symlink.
```

Comment:
Please follow project convention of using tabs.

Reply:
`make compliant` did not complained about this one for some reason. Fixed.